src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Cms.Module/Drivers/MyTestPartDisplayDriver.cs

```diff
@@ -19,42 +19,34 @@ public MyTestPartDisplayDriver(IContentDefinitionManager contentDefinitionManage
             _contentDefinitionManager = contentDefinitionManager;
         }
 
-        public override IDisplayResult Display(MyTestPart MyTestPart)
+        public override IDisplayResult Display(MyTestPart part, BuildPartDisplayContext context)
         {
-            return Combine(
-                Initialize<MyTestPartViewModel>("MyTestPart", m => BuildViewModel(m, MyTestPart))
-                    .Location("Detail", "Content:20"),
-                Initialize<MyTestPartViewModel>("MyTestPart_Summary", m => BuildViewModel(m, MyTestPart))
-                    .Location("Summary", "Meta:5")
-            );
+            return Initialize<TestPartViewModel>(GetDisplayShapeType(context), m => BuildViewModel(m, part, context))
```

Comment:
```suggestion
            return Initialize<MyTestPartViewModel>(GetDisplayShapeType(context), m => BuildViewModel(m, part, context))
```

Reply:
Sorry I read this and I still don't understand why the view model type has changed? Are there two viewmodels? one for edit, one for display, or something?